src/ssh/azext_ssh/aaz/latest/hybrid_connectivity/endpoint/service_configuration/_create.py

```diff
@@ -0,0 +1,257 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+class Create(AAZCommand):
+    """Create a service in serviceConfiguration for the endpoint resource.
+    """
```

Comment:
Please add examples for commands

Reply:
Sorry, it's my mistake. If this comment not registered, then you don't need to add examples.